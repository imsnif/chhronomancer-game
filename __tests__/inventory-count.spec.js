import 'react-native'
import React from 'react'
import InventoryCount from '../components/inventory-count'
import renderer from 'react-test-renderer'

test('InventoryCount (default props) => renders correctly', () => {
  const tree = renderer.create(
    <InventoryCount />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('InventoryCount (with count) => renders correctly', () => {
  const tree = renderer.create(
    <InventoryCount count={1} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
