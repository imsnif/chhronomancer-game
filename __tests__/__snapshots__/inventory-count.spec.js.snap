// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InventoryCount (default props) => renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "flex": 1,
        "flexDirection": "row",
        "fontFamily": "telegrama_raw",
        "fontSize": 8,
        "marginTop": 20,
      }
    }
  >
    Artifacts: 
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "flex": 1,
        "flexDirection": "row",
        "fontFamily": "telegrama_raw",
        "fontSize": 8,
        "marginTop": 20,
      }
    }
  >
    /7
  </Text>
</View>
`;

exports[`InventoryCount (with count) => renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "flex": 1,
        "flexDirection": "row",
        "fontFamily": "telegrama_raw",
        "fontSize": 8,
        "marginTop": 20,
      }
    }
  >
    Artifacts: 
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "flex": 1,
        "flexDirection": "row",
        "fontFamily": "telegrama_raw",
        "fontSize": 8,
        "marginTop": 20,
      }
    }
  >
    1
    /7
  </Text>
</View>
`;
